src/components/Icons/Svg/FT/DAI.tsx

```diff
@@ -8,28 +8,28 @@ export function DAI(props: SVGProps) {
   const { size, ...rest } = props;
   return (
     <svg
-      width={size ?? 44}
-      height={size ?? 44}
-      viewBox="0 0 32 32"
+      width={size ?? 20}
+      height={size ?? 20}
+      viewBox="0 0 20 20"
```

Comment:
for info: you can change width and height without changing viewBox. If you change viewBox I think you need to update all the svg

Reply:
yes in this pr all the SVG's were update, the viewbox you see here is default svg viewbox (not manually changed)